src/main/resources/standardTestCases/backend.test

```diff
@@ -0,0 +1,17 @@
+//Test against a different functional backend in the future
+
+name: backend
+testFormat: workflowsuccess
+
+files {
+  wdl: hello/hello.wdl
+  inputs: hello/hello.inputs
+  options: backend/backend.options
+}
+
+metadata {
+  workflowName: hello
+  status: Succeeded
+  "calls.hello.hello.executionStatus": Done
+  "calls.hello.hello.backend": "Local"
+}
```

Comment:
Actually, apply this comment to all the files with a 
<img width="24" alt="screen shot 2016-06-15 at 11 09 33 am" src="https://cloud.githubusercontent.com/assets/13006282/16085316/add55e10-32e9-11e6-84ea-e92d41bc3a4b.png"> at the bottom